macros/utils/table_operations/get_relation_max_length.sql

```diff
@@ -0,0 +1,16 @@
+{% macro get_relation_max_name_length() %}
+    {{ return(adapter.dispatch('get_relation_max_name_length', 'elementary')()) }}
+{% endmacro %}
+
+{# Snowflake and BigQuery #}
+{% macro default__get_relation_max_name_length(temporary, relation, sql_query) %}
+    {{ return(none) }}
+{% endmacro %}
+
+{% macro redshift__get_relation_max_name_length(temporary, relation, sql_query) %}
+    {{ return(127) }}
```

Comment:
Is it hardcoded in dbt as well? 
Just asking so we could depend on the adapter and not maintain this